providers/base/bin/wifi_nmcli_test.py

```diff
@@ -27,7 +27,7 @@
 
 def legacy_nmcli():
     cmd = "nmcli -v"
-    output = sp.check_output(cmd, shell=True)
+    output = sp.check_output(cmd.split())
```

Comment:
suggest we split the command with shlex library
```suggestion
    output = sp.check_output(shlex.split(cmd))
```